tools/sylph/sylph.xml

```diff
@@ -0,0 +1,392 @@
+                #else:
+                    && ln -s "$database_select.metadata" "database.tsv.gz"
+                #end if
+                && python "$__tool_directory__/sylph_to_taxprof.py" -s output.tsv -m database.tsv.gz -o metaphlan_
+                #if "krona" in $function.outputs:
+                    && python "$__tool_directory__/sylphformatoutput.py" format_for_krona --metaphlan_output *.sylphmpa --krona_output krona.tsv
+                    && mkdir krona_out && mv *krona.tsv krona_out
+                #end if
+                && mkdir metaphlan_out && mv *.sylphmpa metaphlan_out
+            #end if
+    ##OPTION FOR QUERY
+        #else
+            sylph query
+            -t \${GALAXY_SLOTS:-4}
+            input.syldb
+            sylph_sketches/*.sylsp
+            -o output.tsv
+        #end if
+    ]]></command>
+    <inputs>
+        <conditional name="sketch">
+            <param name="type" type="select" label="Select the type of reads used">
+                <option value="single">Individual single-end reads</option>
+                <option value="single_group">Group of single-ended reads</option>
+                <option value="paired">One set of paired-end reads</option>
+                <option value="paired_group"> Group of paired-ended reads</option>
+            </param>
+        <!-- Only permitting fastq as tool input only allows fastq and fastq.gz as file ext -->
+            <when value="single">
+                <param name="input" type="data" format="fastq,fastq.gz,fastqsanger,fastqsanger.gz" label="Single-end input reads"/>
+            </when>
+            <when value="single_group">
+                <param name="input" type="data" format="fastq,fastq.gz,fastqsanger,fastqsanger.gz" label="Single-end input reads" multiple="true"/>
+            </when>            
+            <when value="paired">
+                <param name="input_1" type="data" format="fastq,fastq.gz,fastqsanger,fastqsanger.gz" label="Paired-end input reads 1"/>
+                <param name="input_2" type="data" format="fastq,fastq.gz,fastqsanger,fastqsanger.gz" label="Paired-end input reads 2"/>
+            </when>
+            <when value="paired_group">
+                <param name="input" type="data_collection" format="fastq,fastq.gz,fastqsanger,fastqsanger.gz" label="Paired-end input reads" collection_type="paired"/>
+            </when>
+        </conditional>
+        <conditional name="function" >
+            <param name="functions" type="select" label="ANI querying or taxonomic profiling" help="ANI(Average Nucleotide Identity) is a measure of nucleotide-level similarity between the genomes of two microogranisms. Profile querying involves comparing functional or sequence profiles derived from the genomic data.">
+                <option value="profile">Taxanomic Profile</option>
+                <option value="ani">ANI Query</option>
```

Comment:
I am wondering if it would not more sense to have 2 Galaxy tools (one for `sylph profile`, one for `sylph query`).
That would be closer of how people would use it via command-line. What do you think?

Reply:
@tcollins2011 any opinion here?